test/e2e/density.go

```diff
@@ -181,6 +191,26 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			additionalNameStr := strconv.Itoa(minionCount) + "-" + string(util.NewUUID())
+			additionalRCName = "my-hostname-density" + additionalNameStr
+			Logf("Schedling additional Pods to measure startup latencies")
+			config = RCConfig{Client: c,
+				Image:         "gcr.io/google_containers/pause:go",
+				Name:          additionalRCName,
+				Namespace:     ns,
+				PollInterval:  itArg.interval,
+				PodStatusFile: fileHndl,
+				Replicas:      minionCount,
+			}
+			latencies, err := RunRC(config)
+			expectNoError(err)
+			sort.Float64s(latencies)
+			perc50 := latencies[len(latencies)/2]
+			perc90 := latencies[(len(latencies)*9)/10]
+			perc99 := latencies[(len(latencies)*99)/100]
+			Logf("Latencies: %v", latencies)
```

Comment:
I don't think (latencies[len(latencies)-1]/minionCount) is the thing you want. It's longest latency divided by number of Pods, which is not really interesting. I can add some kind of throughput logging, but I'm not really sure what kind of throughput to measure. I can sum all latencies and have minionCount/sum as a proxy for "sequential" throughput (if that's interesting), or measure walltime for parallel start for "real-ish" throughput. The second thing is already logged for cluster saturation phase, so I don't know how important is second measurement.